frontend/src/lib/components/LemonInput/LemonInput.tsx

```diff
@@ -32,7 +28,7 @@ interface LemonInputPropsBase
 }
 
 interface LemonInputPropsText extends LemonInputPropsBase {
-    type?: 'text' | 'email'
+    type?: 'text' | 'email' | 'search'
```

Comment:
I might be missing something but doesn't look like `type="search"` does anything?

Reply:
Oh, I totally forgot there is a `search` HTML `type`. In this case, I think Chris has a good point in https://github.com/PostHog/posthog/pull/11179#issuecomment-1209646799 that we might want to simply rely on this `type` value instead of adding a component (i.e. if it ha `type="search"` and no `icon`, it should have the magnifier icon).

Reply:
Sounds good. Tried that out instead. Feels a little less clean but from an external interface point of view makes sense